benchmarks/benchmarks/optimize.py

```diff
@@ -209,7 +209,8 @@ def bench_run(self, x0, methods=None, **minimizer_kwargs):
         if methods is None:
             methods = ["COBYLA", 'Powell',
                        'L-BFGS-B', 'BFGS', 'CG', 'TNC', 'SLSQP',
-                       "Newton-CG", 'dogleg', 'trust-ncg', 'trust-region-exact']
+                       "Newton-CG", 'dogleg', 'trust-ncg', 'trust-trlib',
```

Comment:
If think that `trlib-krylov` is a generic name that would apply to any of the methods Steihaug Toint, Sequential Subspace Method (Hager 1999), IP-SSM (Erway et al. 2009) and the method implemented here by Nick Gould which all have different behaviour. For these reason I think it is more appealing to stick with the original name `trust-trlib`, while it the name does not tell much how it works, you can find as one of the first google hits for `trlib` a detailed description of it in the preprint [1].

1: http://www.optimization-online.org/DB_HTML/2016/11/5724.html